<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1585553199255' stop='1585553199289'>
    <name>Second Pull</name>
    <title>Second Pull</title>
    <test-cases>
        <test-case start='1585553199255' status='failed' stop='1585553199259'>
            <name>Add to Comparison</name>
            <title>Add to Comparison</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1585553199259' status='failed' stop='1585553199264'>
            <name>Remove from Comparison</name>
            <title>Remove from Comparison</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1585553199265' status='failed' stop='1585553199270'>
            <name>Sort in Market</name>
            <title>Sort in Market</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1585553199270' status='failed' stop='1585553199275'>
            <name>Sort by tag</name>
            <title>Sort by tag</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1585553199275' status='failed' stop='1585553199281'>
            <name>Yandex Music</name>
            <title>Yandex Music</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1585553199281' status='failed' stop='1585553199287'>
            <name>Play Music</name>
            <title>Play Music</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393</message>
                <stack-trace>Error: EADDRINUSE connect EADDRINUSE 127.0.0.1:63393
                    at ClientRequest.&lt;anonymous> (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\selenium-webdriver\http\index.js:238:15)
                    at ClientRequest.emit (events.js:210:5)
                    at Socket.socketErrorListener (_http_client.js:406:9)
                    at Socket.emit (events.js:210:5)
                    at emitErrorNT (internal/streams/destroy.js:91:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
                    at processTicksAndRejections (internal/process/task_queues.js:80:21)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:8:5)
                    at addSpecsToSuite (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Alexey\automationJS\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\Alexey\automationJS\tests\secondPull.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:945:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
                    at Module.load (internal/modules/cjs/loader.js:798:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:711:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>